ansible/playbooks/AdoptOpenJDK_AIX_Playbook/roles/yum/tasks/main.yml

```diff
@@ -120,14 +119,29 @@
     - sudo
     - tar
     - tcl
-    - tk
     - unzip
     - wget
     - xz-libs
     - zip
     - zsh
   tags: yum
 
+- name: Remove packages - in case installed historically, or by accident
+  yum: name={{ item }} state=absent
+  with_items:
+    - libXrender-devel
+    - libXrender
+    - libXft
+    - tk
+    - renderproto
+  tags: yum
```

Comment:
I don't want anything in place that could potentially damage a users' machine, so can we add the `adoptopenjdk`  tag to this so it is skipped when an end user uses the playbooks in the expected way please? I'm templted to suggest just running this as a one-off on all our machines rather than put it in here, but I'm ok with it satying too :-)

Reply:
OK, I added the tag - BUT - confused, because `adoptopenjdk` is not used anywhere else in this playbook. IMHO, the additional tag is misplaced. IMHO, if you are going to use this tag - that is a separate PR where either all playbooks are re-tagged, or a separate PR per playbook where re-tagging occurs.

Only adding it here, for these (two) plays adds confusion, imho.

And, for the record, there are likely several `stray` RPMs that are installed to satisfy a dependency - that was never needed. (e.g., `expect`, `tk` and `tcl` so that the yum.sh script can run).